### PR TITLE
fix homepage & description in scikit-image easyconfigs

### PR DESCRIPTION
**Pull request by boegel**

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 13 out of 16 (16 easyconfigs in this PR)
node2102.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/be5a3d0f26db9a73f7b58ede6cef4f65 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 16 out of 16 (16 easyconfigs in this PR)
node2102.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/4549570f5704a58bb009ed24152c1b54 for a full test report.

**Comment by boegel** [comment]
@bartoldeman Adding `dask` as dependency is required to avoid that it gets auto-downloaded...

**Comment by akesandgren** [comment]
Test report by @akesandgren
**FAILED**
Build succeeded for 46 out of 65 (14 easyconfigs in this PR)
b-an03.hpc2n.umu.se - Linux ubuntu 16.04, Intel(R) Xeon(R) CPU E5-2690 v4 @ 2.60GHz, Python 2.7.12
See https://gist.github.com/de667d3247630cd3cbf23c5751d3f0bf for a full test report.

**Comment by boegel** [comment]
@akesandgren `Segmentation fault` during `libjpeg-turbo-1.5.1-intel-2016b.eb` & other `intel/2016b` builds is probably because of https://github.com/easybuilders/easybuild-easyconfigs/issues/3646?

Certainly not related to the changes made in this PR...

**Comment by akesandgren** [comment]
Ah, that old problem. Hadn't fixed that in my testreport environment. Redoing stuff....

File: easybuild/easyconfigs/s/scikit-image/scikit-image-0.13.0-foss-2017b-Python-3.6.3.eb [hunk 1]
```diff
@@ -19,12 +16,7 @@ dependencies = [
     ('Pillow', '4.3.0', versionsuffix),
 ]
```
Comment (akesandgren) [inline easybuild/easyconfigs/s/scikit-image/scikit-image-0.13.0-foss-2017b-Python-3.6.3.eb]: Isn't dask a dependency for this version?

Comment (akesandgren) [reply easybuild/easyconfigs/s/scikit-image/scikit-image-0.13.0-foss-2017b-Python-3.6.3.eb]: Same question for the rest of the diff

Comment (boegel) [reply easybuild/easyconfigs/s/scikit-image/scikit-image-0.13.0-foss-2017b-Python-3.6.3.eb]: Good point, fix included in 45b58458

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 22 out of 22 (22 easyconfigs in this PR)
node2065.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/613d972cec1309d24b64b3f41c60f0bf for a full test report.

**Comment by akesandgren** [comment]
Going in, thanks @boegel!

**Comment by akesandgren** [comment]
Going in, thanks @boegel!

**Pull request merged**